### PR TITLE
Setup and Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by darikmohammed**
# Work Description
>  create an HTML and CSS section for the toolbar (or header) and the headline section. 
Screen Shot 2022-04-21 at 1 37 05 AM

## Tasks done.
- setup html-css linter 
- add HTML and CSS code for the header section
- add HTML and CSS for the headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          <img src="./icons/mobile-menu-btn.svg" alt="mobile menu" />
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="headline">
+        <div class="primary-headline">
+          <h1>
+            Hey there. I'm Darik
+            <span class="primary-color">I'm a software developer</span>
```
Comment (aimemalaika) [inline index.html:26]: - [ ] Kindly put the subtitle uses a header tag hx (examples: <h1>,<h2>,<h3>, etc. not in a `span` inside an h1 tag

Comment (aimemalaika) [inline index.html:26]: - [ ] Kindly put the subtitle uses a header tag hx (examples: `h1`,`h2`,`h3`, etc. not in a `span` inside an h1 tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-container">
+        <div class="logo">
+          <a href="#" id="welcome-logo">Welcome</a>
+        </div>
+        <nav class="mobile-menu-btn">
+          <img src="./icons/mobile-menu-btn.svg" alt="mobile menu" />
```
Comment (aimemalaika) [inline index.html:17]: - [ ] Kindly set hamburger as button inside the nav tag not an image

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @darikmohammed , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (darikmohammed) [reply index.html:26]: @aimemalaika while seeing through the Figma design it is labeled that the word Hey there. I'm Darik. I'm a software developer is set in the same header. Do I have to make "I'm a software developer" an h2 with the same size as "Hey there. I'm Darik"?

**Review by topeogunleye — APPROVED** [review]
### Hi @darikmohammed, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**